EasyLearn/app/views/settings/index.html.erb

```diff
@@ -0,0 +1,97 @@
+
+<head>
+  <meta charset = "UTF-8">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="stylesheet" href="app/assets/stylesheets/styles.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap" rel="stylesheet">
+  <title>Settings</title>
+</head>
+<body>
```

Comment:
The head and body tags are not needed in this file, since they are inherited from the application.html.erb file.

Do you occupy all these libraries only in this file or in several files?